examples/main/main.spec.js

```diff
@@ -42,11 +47,225 @@ describe('examples/main', () => {
+    })
+
+    it('should render section hyperlink', () => {
+      expect($('section p').text())
+        .to.equal(String("https://makethislink.com"));
+    })
+
+    it('should render all blogs into blog container', () => {
+      let blogs = [
+        '..',
+        'my-first-blog',
+        'my-first-blog',
+      ]
+
+      $('#blog-container span').each(function (index, element) {
+        expect($(this).html()).to.equal(blogs[index]);
+      });
+    })
+  })
+
+
+  //new blog example  
+  describe('new-blog/example.html', () => {
+    let $;
+    before(() => {
+      $ = getSelector(path.join(__dirname, 'dist', 'new-blog', 'example.html'))
+    })
+
+    it('should render html body text', () => {
+      expect($('body').text().trim())
+        .to.equal(String("new-blog"));
+    })
+  })
+
+  //new blog
+  describe('new-blog/index.html', () => {
+    let $;
+    before(() => {
+      $ = getSelector(path.join(__dirname, 'dist', 'new-blog', 'index.html'))
+    })
+
+    it('should render all blogs into blog container', () => {
+      let blogs = [
+        '..',
+        'new-blog',
+        'new-blog',
+      ]
+
+      $('#blog-container span').each(function (index, element) {
+        expect($(this).html()).to.equal(blogs[index]);
+      });
+    })
+    it('should render header text', () => {
+      expect($('body main section #newblog').html())
+        .to.equal(String("new-blog"));
+    })
+    it('should render first para text', () => {
+      expect($('body main section p').first().html())
+        .to.equal(String(".."));
+    })
+    it('should render global meta site name', () => {
+      expect($('body main section p').last().html())
+        .to.equal(String("{{ globalMeta.siteName }}"));
+    })
+
+  })
+
+  //new blog/sub blog/example  
+  describe('new-blog/sub-blog/example.html', () => {
+    let $;
+    before(() => {
+      $ = getSelector(path.join(__dirname, 'dist', 'new-blog', 'sub-blog', 'example.html'))
+    })
+
+    it('should render html body text', () => {
+      expect($('body').text().trim())
+        .to.equal(String(`new-blog${path.sep}sub-blog`));
+    })
+  });
+
+
+  describe('new-blog/sub-blog/index.html', () => {
+    let $;
+    before(() => {
+      $ = getSelector(path.join(__dirname, 'dist', 'new-blog', 'sub-blog', 'index.html'))
+    })
+
+    it('should render all blogs into blog container', () => {
+      let blogs = [
+        `..${path.sep}..`,
+        `new-blog${path.sep}sub-blog`,
+        'sub-blog',
+      ]
+
+      $('#blog-container span').each(function (index, element) {
+        expect($(this).html()).to.equal(blogs[index]);
+      });
+    })
+    it('should render header text', () => {
+      expect($('body main section #inside-the-new-blog').html())
+        .to.equal(String("Inside the new blog"));
+    })
+    it('should render first para text', () => {
+      expect($('body main section p').first().html())
+        .to.equal(String("...."));
```

Comment:
The issue is here I believe

Reply:
It is supposed to be `.to.equal(`..${path.sep}..`)`

Reply:
Yes but I have seen that in build HTML file.It shows me like `....` .Do you know why this will happen?

Reply:
In my build, it shows `../..` :/

Reply:
Oh got it! It's because of the `String()` wrapper. In windows it probably becomes `String(..\..)` and `\` escapes in String.

We don't need that `String()` there

Reply:
Still the same issue :(

Reply:
Now adding path.sep is suppose to solve this